### PR TITLE
Add a whats included/features section to docs

### PR DESCRIPTION
**Pull request by madhuravi**
I was inspired by some other docs to have this in the introduction page. It gives a concise view of what UberFx can offer for users.

### CODE REVIEW

**Comment by madhuravi** [comment]
@uber-go/service-framework @Lupie

File: README.md [hunk 1]
```diff
@@ -5,14 +5,22 @@
 [![Build Status][ci-img]][ci]
 [![Report Card][report-card-img]][report-card]
 
-## Status
+UberFx is a flexible, modularized framework for building robust and performant
+services. It removes boilerplate code out of your hands and lets you focus on
```
Comment (Lupie) [inline README.md]: It's awkward to say "removes…out of your hands." Some alternatives:
- It removes boilerplate from your to-do list and lets you focus on
- It removes boilerplate from your job, letting you focus on
- It takes care of the boilerplate code and lets you focus on

Feel free to mix and match.

File: README.md [hunk 2]
```diff
@@ -5,14 +5,22 @@
...
 
-Pre-ALPHA. API Changes are **highly likely**.
+## Status
 
-## Abstract
+Alpha. Expect minor API changes and bug fixes. (Beta release coming soon)
 
-This framework is a flexible, modularized basis for building robust and
-performant services at Uber with the minimum amount of developer code.
+## What's included
```
Comment (Lupie) [inline README.md]: This section is wonderful.

Something to think about: UberFx doesn't just _include_ these things. It actually gets them all started for you and plops them into your project. That's awesome.

For people who haven't used this, do we want stronger language, like "What's included" as a heading, with a sentence below, like "UberFx builds the following into your new service:" to be extra clear?

File: README.md [hunk 3]
```diff
@@ -5,14 +5,22 @@
...
+* Configuration, logging, metrics and tracing setup by default
```
Comment (Lupie) [inline README.md]: metrics, and

File: README.md [hunk 4]
```diff
@@ -5,14 +5,22 @@
...
+* Configuration provider that seamlessly merges static and dynamic config
+* Application-level as well as runtime metrics for effective monitoring
+* Context aware logging for easy debugging
```
Comment (Lupie) [inline README.md]: Context-aware

File: README.md [hunk 5]
```diff
@@ -5,14 +5,22 @@
...
+* RPC module with thrift interfaces for microservices
```
Comment (Lupie) [inline README.md]: Thrift with a capital T

File: README.md [hunk 6]
```diff
@@ -5,14 +5,22 @@
...
+* HTTP module with intelligent defaults for web applications
```
Comment (glibsm) [inline README.md]: what's an example of an intelligent default?

File: README.md [hunk 7]
```diff
@@ -5,14 +5,22 @@
...
+* Async task module for executing async tasks durably
```
Comment (glibsm) [inline README.md]: `Durable async task module`

File: README.md (hunk 2 above)
Comment (madhuravi) [inline README.md]: good idea.

File: README.md (hunk 3 above)
Comment (madhuravi) [inline README.md]: why?

File: README.md (hunk 6 above)
Comment (madhuravi) [inline README.md]: tracing, auth, panic handling etc.

File: README.md (hunk 7 above)
Comment (madhuravi) [inline README.md]: It's in the latter half of the sentence. Would you prefer just "Durable async task module" I added the for.. part for emphasis.

File: README.md [hunk 8]
```diff
@@ -5,14 +5,23 @@
...
 
-## Abstract
+Alpha. Expect minor API changes and bug fixes. (Beta release coming soon)
 
-This framework is a flexible, modularized basis for building robust and
-performant services at Uber with the minimum amount of developer code.
+## What's included
+UberFx builds the following into your service:
+* Configuration, logging, metrics, and tracing setup by default
+* Configuration provider that seamlessly merges static and dynamic config
```
Comment (sectioneight) [inline README.md]: weird that configuration is the first word in the first two bullet points?

**Pull request merged**